app/View/GalaxyClusters/add.ctp

```diff
@@ -32,12 +32,14 @@
                     'field' => 'distribution',
                     'options' => $distributionLevels,
                     'default' => isset($cluster['GalaxyCluster']['distribution']) ? $cluster['GalaxyCluster']['distribution'] : $initialDistribution,
-                    'stayInLine' => 1
+                    'stayInLine' => 1,
+                    'type' => 'dropdown'
                 ),
                 array(
                     'field' => 'sharing_group_id',
                     'options' => $sharingGroups,
-                    'label' => __("Sharing Group")
+                    'label' => __("Sharing Group"),
+                    'type' => 'dropdown'
                 ),
                 !isset($forkedClusterMeta) ? '' : sprintf('<div id="fork_galaxy_preview" class="panel-container fork-cluster-preview"><h5>%s %s</h5>%s</div>',
                     __('Forked Cluster data'),
```

Comment:
Remove
https://github.com/MISP/MISP/blob/7736b08f2fd46fbe827258bb2de412321048bb68/app/View/GalaxyClusters/add.ctp#L81 or set it to be the same as
https://github.com/MISP/MISP/blob/7736b08f2fd46fbe827258bb2de412321048bb68/app/View/Elements/genericElements/Form/fieldScaffold.ctp#L45